src/main/java/com/streamforge/realm/twitch/oauth/dtos/TwitchUserTokenDto.java

```diff
@@ -0,0 +1,44 @@
+package com.streamforge.realm.twitch.oauth.dtos;
+
+import java.util.List;
+
+public class TwitchUserTokenDto {
+    private String access_token;
+    private String refresh_token;
+    private Long expires_in;
+    private List<String> scope;
+
+    public TwitchUserTokenDto() { }
+
+    public void setScope(List<String> scope) {
+        this.scope = scope;
+    }
+
+    public List<String> getScope() {
+        return scope;
+    }
+
+    public String getAccess_token() {
+        return access_token;
+    }
+
+    public Long getExpires_in() {
+        return expires_in;
+    }
+
+    public String getRefresh_token() {
+        return refresh_token;
+    }
+
+    public void setAccess_token(String access_token) {
```

Comment:
Is it possible to return instance after setting value (return this) to simplify our code?

Reply:
Oh, there is a bottleneck here, that I really wish to discuss. Let`s talk about this one in private mode.